python/python-psi-impl/src/com/jetbrains/python/codeInsight/PyDataclasses.kt

```diff
@@ -24,9 +24,13 @@ import com.jetbrains.python.psi.types.TypeEvalContext
 const val DATACLASSES_INITVAR_TYPE: String = "dataclasses.InitVar"
 const val DUNDER_POST_INIT: String = "__post_init__"
 const val DUNDER_ATTRS_POST_INIT: String = "__attrs_post_init__"
+const val DUNDER_PYDATNIC_POST_INIT_POST_PARSE: String = "__post_init_post_parse__"
```

Comment:
Shouldn't this be `DUNDER_PYDANTIC_POST_INIT_POST_PARSE` instead of `DUNDER_PYDATNIC_POST_INIT_POST_PARSE` (the same goes for references of the const val later on)?

Reply:
good catch!
I have fixed the typo. And refactor references.
Thank you.